draft-ietf-shmoo-remote-fee.md

```diff
@@ -137,6 +137,8 @@ However, this document does not comment on the actual payment structure
 of the IETF meeting fee other than the requirement for a free option. The fee payment structure is set the by the IETF LLC such that
 the viability of the IETF and the need of IETF participants to work productively within the IETF can be warranted.
 
+The LLC is responsible to ensure the financial stability of the IETF and therefore should monitor
+trends in the use of the free participating option that could endanger the viability of the IETF.
```

Comment:
I might change "participating" to "participation", but otherwise +1 to this

Reply:
```suggestion
trends in the use of the free participation option that could endanger the viability of the IETF.
```